ros/src/util/packages/runtime_manager/scripts/computing.yaml

```diff
@@ -79,11 +79,53 @@ subs :
         param: icp
         gui  :
           dialog_height : 400
+      - name : ndt_slam
+        desc : ndt_slam
+        cmd  : roslaunch lidar_localizer ndt_slam.launch
+        param: ndt_slam
+        gui  :
+          dialog_width  : 700
+          dialog_height : 550
+          points_queue_size :
+            flags : [ nl ]
+          input_current_pose_topic :
+            flags : [ nl ]
+            depend      : use_fusion_localizer_feedback
+            depend_bool : 'lambda v : v == 1'
+          world_frame :
+            flags : [ nl ]
+          init_x  :
+            user_category : ''
+          init_yaw:
+            flags : [ nl ]
+          max_iterations:
+            flags : [ nl ]
+          save_map_leaf_size :
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+          min_scan_range :
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+          max_scan_range :
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+          min_add_scan_shift:
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+          separate_mapping :
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+          separate_map_size :
+             depend      : with_mapping
+             depend_bool : 'lambda v : v == 1'
+             flags : [ nl ]
+          use_nn_point_z_when_initial_pose :
+             flags : [ nl ]
     - name : fusion_localizer
       desc : fusion_localizer desc sample
       subs :
       - name : ekf_localizer
-        desc : pose and twist fusion for robust localization 
+        desc : pose and twist fusion for robust localization
```

Comment:
why are you changing this? it is unrelated

Reply:
@amc-nu Thank you!
It seems that atom of my editor was set to delete extra whitespace at the end of the line when saving the file.